workers/unity/Assets/Playground/Scripts/Worker/GameLogicWorkerConnector.cs

```diff
@@ -14,7 +16,27 @@ public class GameLogicWorkerConnector : DefaultWorkerConnector
         private async void Start()
         {
             Application.targetFrameRate = 60;
-            await Connect(WorkerUtils.UnityGameLogic, new ForwardingDispatcher()).ConfigureAwait(false);
+
+            var connParams = new ConnectionParameters
+            {
+                WorkerType = WorkerUtils.UnityGameLogic,
+                DefaultComponentVtable = new ComponentVtable(),
+                Network = new NetworkParameters
+                {
+                    UseExternalIp = UseExternalIp
```

Comment:
this will be overwritten when `resolve` is called, right?

Reply:
It can be